googleapis/google-cloud-node
# migrate code from googleapis/nodejs-essential-contacts

- initial generation of template files
- remove extraneous files
- remove extraneous files
- feat: initial generation of tests and samples (#2)
- chore: release 1.0.0 (#5)
- chore(deps): update dependency mocha to v8 (#6)
- Update dependency sinon to v11 (#7)
- fix: GoogleAdsError missing using generator version after 1.3.0 (#3)
- chore: release 1.0.1 (#13)
- build: edit package name to match published version (#16)
- fix: make request optional in all cases (#17)
- chore: release 1.0.2 (#18)
- fix(deps): google-gax v2.17.0 with mTLS (#22)
- chore: release 1.0.3 (#23)
- fix(deps): google-gax v2.17.1 (#26)
- chore: release 1.0.4 (#27)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#28)
- chore: release 1.0.5 (#29)
- chore: release 1.0.5 (#30)
- chore(nodejs): update client ref docs link in metadata (#36)
- fix(build): migrate to using main branch (#38)
- fix(deps): google-gax v2.24.1 (#39)
- chore: release 1.0.6 (#40)
- feat: turns on self-signed JWT feature flag (#41)
- chore: release 1.1.0 (#42)
- fix: flag module as GA (#44)
- chore: release 1.1.1 (#45)
- chore(deps): update dependency mocha to v9 (#47)
- build(node): update deps used during postprocessing (#1243) (#54)
- chore: fix the wrong post processor image (#56)
- chore(deps): update dependency @types/node to v16 (#55)
- chore(cloud-rad): delete api-extractor config (#57)
- chore(deps): update dependency sinon to v12 (#58)
- docs(samples): add example tags to generated samples (#59)
- build: add generated samples to .eslintignore (#60)
- docs(node): support "stable"/"preview" release level (#1312) (#63)
- chore: add api_shortname and library_type to repo metadata (#62)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#65)
- test(nodejs): remove 15 add 16 (#1322) (#67)
- build: update copyright year to 2022 (#69)
- chore: update v2.12.0 gapic-generator-typescript (#71)
- chore(deps): update dependency sinon to v13 (#73)
- docs(nodejs): version support policy edits (#1346) (#75)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#76)
- chore: update v2.14.2 gapic-generator-typescript (#80)
- build(node): update client library version in samples metadata (#1356) (#87)
- chore(deps): update dependency sinon to v14 (#90)
- build!: update library to use Node 12 (#92)
- chore(main): release 2.0.0 (#93)
- chore(deps): update dependency jsdoc-region-tag to v2 (#97)
- chore(deps): update dependency jsdoc-fresh to v2 (#96)
- fix(docs): describe fallback rest option (#98)
- chore(main): release 2.0.1 (#99)
- chore(deps): update dependency linkinator to v4 (#104)
- fix: better support for fallback mode (#105)
- fix: change import long to require (#106)
- fix: do not import the whole google-gax from proto JS (#1553) (#109)
- fix: allow passing gax instance to client constructor (#110)
- fix: preserve default values in x-goog-request-params header (#111)
- test: use fully qualified request type name in tests (#113)
- fix(deps): use google-gax v3.5.2 (#117)
- chore(deps): update dependency @types/node to v18 (#115)
- fix: regenerated protos JS and TS definitions (#120)
- chore(deps): update dependency jsdoc to v4 (#119)
- chore(main): release 2.0.2 (#107)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
